internal_testsuites/golang/testsuite/startosis_run_sh_task_test/run_task_sh_task_test.go

```diff
@@ -17,12 +17,13 @@ def run(plan):
 `
 	runshStarlarkFileArtifact = `
 def run(plan):
-  result = plan.run_sh(run="mkdir -p src && echo kurtosis > /task/src/tech.txt", store=["src/tech.txt", "/task/src"])
-  file_artifacts = result.file_artifacts
-  result2 = plan.run_sh(run="cat /src/temp/tech.txt", files={"temp": file_artifacts[0]}, workdir="src")
+  result = plan.run_sh(run="mkdir -p src && echo kurtosis > /src/tech.txt", store=["/src/tech.txt", "/src"])
```

Comment:
can we add a test for a custom image? I ask that because thats where I found the bug

Reply:
ok; will do in next pr - maybe just use the eth image? @h4ck3rk3y

Reply:
can we add it to this PR itself as this PR is fixing the bug that test found?